cli/package.json

```diff
@@ -55,7 +55,7 @@
     "minimist": "^1.2.6",
     "ospath": "^1.2.2",
     "pretty-bytes": "^5.6.0",
-    "proxy-from-env": "1.0.0",
+    "proxy-from-env": "1.1.0",
```

Comment:
@stefdev49 can you update the yarn lock as well?

Reply:
I'm looking at 1.1.0 to try & determine what changed between 1.0.0 and 1.1.0 and this might be the reason we are still on 1.0.0....https://github.com/Rob--W/proxy-from-env/issues/13

@flotwig Do you remember the impact of this change or see any issues with updating?

Reply:
Cypress treats `x_PROXY` with higher precedence than `npm_config_x_proxy` because it's more "local" than npm config. proxy-from-env after 1.0.0 changes that precedence to prefer `npm_config_x_proxy` vars over `x_PROXY` vars. Updating proxy-from-env would be breaking for this reason.

Reply:
Since this provides better support, we likely havea solid use-case for introducing a new patch-package so we can maintain our level of prescience while also leveraging support for npmrc values as well..We already have this setup in the repo, you'd just need to follow the directions outlined here: https://github.com/ds300/patch-package#making-patches

Mind giving it a shot and verify if this still resolves the issues you are seeing experiencing with your company's proxy setup.

Reply:
Oh actually, patch-package won't work here because the CLI is distributed as a node_module. So we'd either have to fork proxy-from-env or inline it in the CLI.

Reply:
@flotwig We are using the module directly, not the CLI so wouldn't patch-package work fine?

Reply:
Never mind. 🤦🏻‍♀️ . I thought you meant using the dependency’s cli not that this is being pulled into the Cypress CLI. Yeah, patch-package not an option here. I personally think pulling the code forward from `proxy-from-evn` would be the better approach here rather than forking a maintaining an entire repo and needing to release another dependency. It's pretty light weight.